CMakeLists.txt

```diff
@@ -167,6 +176,11 @@ macro(maketestcpp fname)
     target_link_libraries(${fname} PRIVATE rtosc-cpp rtosc)
     #add_test(memcheck_${fname} valgrind --leak-check=full --show-reachable=yes --error-exitcode=1 ./${fname})
 endmacro(maketestcpp)
+if(PKG_CONFIG_FOUND)
+    set(RTOSC_LIBLO_LIBRARIES "${LIBLO_LIBRARIES}")
+else()
+    set(RTOSC_LIBLO_LIBRARIES "liblo::liblo")
```

Comment:
I'm guessing this should only trigger for MSVC, but the specific if() condition has me second guessing myself.

Reply:
Having this for Windows was the main idea back then. On Linux, the CMake way does not seem to work, which is why we have to use pkg-config.

If I would use `if(NOT WIN32)` instead, this would bring weird errors for Linux users without `pkg-config`. However, not testing for Linux (i.e. what we see in this code section) has the potential that a Linux user without `pkg-config` would try CMake, and liblo CMake build is not supported on Linux.

I could either add an additional `if(NOT WIN32)` check, or we leave it like this, assuming that in the future, liblo CMake build might work on Linux?

Reply:
I'd say in that case just a simple comment around line 178 explaining which platforms this should or shouldn't trigger on ought to be sufficient. That way we have the context if modifications are made in the future.